sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/BroadcastExchangeExec.scala

```diff
@@ -90,7 +91,8 @@ case class BroadcastExchangeExec(
 
   override def runtimeStatistics: Statistics = {
     val dataSize = metrics("dataSize").value
-    Statistics(dataSize)
+    val numRows = metrics("numRows").value
+    Statistics(dataSize, Some(numRows))
```

Comment:
It seems the PR description says nothing about this update, so could you describe it there?

Reply:
Updated it to `This pr adds numRows to the metric and runtimeStatistics of BroadcastExchangeExec.`